changelog/2024-10-07-api-changes-v2.mdx

```diff
@@ -40,6 +40,9 @@ On October 7, 2024, the mittwald API underwent several changes, including the re
 
 _Disclaimer: This summary is AI-generated. If you find any discrepancies, please refer to the detailed changes below._
 
+### Developer's note
+
+Even if these changes appear to be breaking, the actual response of the above-mentioned Order routes does not actually change. `first_name` and `last_name` were incorrectly named in the generated spec, because technically `firstName` and `lastName` were already returned beforehand.
```

Comment:
```suggestion
Even if these changes appear to be breaking, the response of the above-mentioned Order routes does not actually change. `first_name` and `last_name` were incorrectly named in the generated spec, because technically `firstName` and `lastName` were already returned beforehand.
```
oder actually weg, aber nicht actual ... actually